data-service/src/test/java/com/amazonaws/blox/dataservice/test/data/ApiModelBuilders.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"). You may
+ * not use this file except in compliance with the License. A copy of the
+ * License is located at
+ *
+ *     http://aws.amazon.com/apache2.0/
+ *
+ * or in the "LICENSE" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amazonaws.blox.dataservice.test.data;
+
+public class ApiModelBuilders {}
```

Comment:
not used class?

Reply:
Ah, hm. I wanted to establish the naming pattern for the Builders we'd use for the `data-service-model` classes. Perhaps a bit premature, will remove.